mu-plugins/central-hub/tests/phpunit/integration/meta-data/saveCustomFields.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Tests for the function save_custom_fields().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Integration\Metadata;
+
+use function spiralWebDB\Metadata\save_custom_fields;
+use function add_post_meta;
+use function get_post_meta;
+use function delete_post_meta;
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+/**
+ * Class Tests_SaveCustomFields
+ *
+ * @package spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @group   meta-data
+ */
+class Tests_SaveCustomFields extends Test_Case {
+
+	/**
+	 * Instance of the post for each test.
+	 *
+	 * @var WP_Post
+	 */
+	protected $post;
+
+	/**
+	 * Prepare the test environment before each test.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		// Create and get the post.
+		$this->post     = self::factory()->post->create_and_get();
+		$this->post->ID = 287;
+	}
+
+	/**
+	 * Clean up the test environment after each test.
+	 */
+	public function tearDown() {
+		parent::tearDown();
+		$config = [];
```

Comment:
@hellofromtonya `$config` here is a local variable, local only to the `tearDown()` method. It has no effect on any of the tests. Therefore, please delete this line.

Reply:
@hellofromtonya You're right. `$_POST` has global scope, but not `$config`. And PHPStorm even flagged it as an unused local variable. 

Deleted.